airgun/entities/contentview_new.py

```diff
@@ -1,11 +1,14 @@
 from navmazing import NavigateToSibling
+from wait_for import wait_for
 
 from airgun.entities.base import BaseEntity
 from airgun.navigation import NavigateStep
 from airgun.navigation import navigator
 from airgun.utils import retry_navigation
 from airgun.views.contentview_new import NewContentViewCreateView
 from airgun.views.contentview_new import NewContentViewTableView
+from airgun.views.contentview_new import NewContentViewEditView
```

Comment:
If this is only Content View and the older one is absolute, I think we can rename this to ContentView instead of adding `New` prefix